codegen-service/llm-generation-core/src/main/java/org/rj/modelgen/llm/prompt/PromptSubstitution.java

```diff
@@ -6,13 +6,21 @@ public class PromptSubstitution {
     private final String existingString;
     private final String newString;
 
+    public getExistingString() {

```

Comment:
Might be some changes that weren't pushed - this & next method should return string

Reply:
Addressed